.github/workflows/pull-request.yml

```
@@ -114,3 +114,57 @@ jobs:
         with:
           name: macOS.zip
           path: macOS
+
+######################
+###### Windows #######
+######################
+
+  windows:
+    name: Windows
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          submodules: true
+
+      # Append PR number to version
+      - name: Append PR number to version
+        shell: bash
+        run: echo -n "-PR#${{ github.event.pull_request.number }}" >> version
```

@brindosch 
Hier die Berichtigung:
```suggestion
        tr -d '\n' < version > temp && mv temp version
        run: echo -n "-PR#${{ github.event.pull_request.number }}" >> version
```